Source/JNA/pom.xml

```diff
@@ -202,6 +202,8 @@
                     <configuration>
                         <!-- Slightly faster builds, see https://issues.apache.org/jira/browse/MCOMPILER-209 -->
                         <useIncrementalCompilation>false</useIncrementalCompilation>
+                        <source>1.8</source>
```

Comment:
One change.  If you go up to properties, you will see the java version settings there.  Those control it across the board, so please set it there.  Otherwise, I'm blown away.  I didn't expect such a quick response or realize just quite how easy this was.  So thank you so very much!

here is what should be changed instead for compiler.

```
        <maven.compiler.source>1.7</maven.compiler.source>
        <maven.compiler.target>1.7</maven.compiler.target>
        <maven.compiler.testSource>1.7</maven.compiler.testSource>
        <maven.compiler.testTarget>1.7</maven.compiler.testTarget>
```

All to 1.8 works.